randovania/games/dread/json_data/Ferenia.txt

```diff
@@ -598,13 +598,13 @@ Extra - asset_id: collision_camera_011
                   Walljump (Advanced) or Use Spin Boost
                   All of the following:
                       # https://youtu.be/5pnZydkRzVM - This branch is relevant when breaking the blocks with Power Bombs
-                      Infinite Bomb Jump (Advanced) and Lay Normal Bomb
+                      Diagonal Bomb Jump (Beginner) and Lay Normal Bomb
           All of the following:
               # https://youtu.be/mLLlXtp9Dn4
-              Infinite Bomb Jump (Expert) and Lay Normal Bomb
+              Diagonal Bomb Jump (Advanced) and Lay Normal Bomb
           All of the following:
-              # This branch uses Cross Bombs both to break the blocks and climb into the tunnel in a similar kind of way as the clip for the Expert Infinite Bomb Jump
-              Infinite Bomb Jump (Intermediate) and Lay Cross Bomb
+              # https://youtu.be/e8qsyZlA8eo
+              Diagonal Bomb Jump (Intermediate) and Lay Cross Bomb
```

Comment:
Is the cross bomb one really harder than the normal bomb one, once the blocks are destroyed?

Reply:
I think I placed them like this primarily off thinking that with normal bombs you have more ability to use sideways mobility to reach into the hole, while with cross bombs you have to be launched more directly upwards. I could be wrong though.